src/__tests__/CompareResults/beta/fetchCompareResults.test.tsx

```diff
@@ -61,9 +65,10 @@ describe('Results View/fetchCompareResults', () => {
       />,
     );
 
-    expect(spyOnFetch).toHaveBeenCalledWith(
+    expect(global.fetch).toHaveBeenCalledWith(
       'https://treeherder.mozilla.org/api/perfcompare/results/?base_repository=mozilla-central&base_revision=6089e7f0fa57a29c6d080f135f65e146c34457d8&new_repository=mozilla-central&new_revision=1d5eb1343cc87a9be3dfe4b884822506ffdda7d3&framework=1&interval=86400&no_subtests=true',
     );
     expect(store.getState().compareResults.data).toStrictEqual({});
+    expect(document.body).toMatchSnapshot();
```

Comment:
(optional) Maybe the snapshot would be easier to read with somethig such as:

```js
expect(await screen.findByTestId("results-table")).toMatchSnapshot();
```
but that's up to you!